bin/setup

```diff
@@ -2,12 +2,21 @@
 
 set -e
 
+gem_install_log="${GEM_INSTALL_LOG_PATH:-.gem-install.log}"
+
+gem_install() {
+  gem_name=$1
+
+  gem install "$gem_name" --user-install &> "$gem_install_log" \
+    && echo "Installed $gem_name"
```

Comment:
Maybe `| tee $gem_install_log` instead of this dance -- then it will go to output in addition to a log. Or not, if you're doing this to keep things pretty!

Reply:
Definitely doing this to keep things pretty!

Reply:
:nail_care: